src/Kobo/DownloadHelper.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace App\Kobo;
+
+use App\Entity\Book;
+use App\Entity\Kobo;
+use App\Exception\BookFileNotFound;
+use App\Kobo\ImageProcessor\CoverTransformer;
+use App\Service\BookFileSystemManager;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class DownloadHelper
+{
+    public function __construct(
+        private readonly BookFileSystemManager $fileSystemManager,
+        private readonly CoverTransformer $coverTransformer,
+        protected UrlGeneratorInterface $urlGenerator,
+        protected LoggerInterface $logger)
+    {
+    }
+
+    protected function getBookFilename(Book $book): string
+    {
+        return $this->fileSystemManager->getBookFilename($book);
+    }
+
+    public function getSize(Book $book): int
+    {
+        return $this->fileSystemManager->getBookSize($book) ?? 0;
+    }
+
+    public function getCoverSize(Book $book): int
+    {
+        return $this->fileSystemManager->getCoverSize($book) ?? 0;
+    }
+
+    public function isEpub3(Book $book): bool
+    {
+        return $book->getExtension() === 'epub3' || $this->readEpubVersionIs3($book) === true;
```

Comment:
?

Reply:
Indeed this is not used anymore, I will remove it

Reply:
done.